runtime/oti/j9.h

```diff
@@ -301,4 +301,12 @@ static const struct { \
 #define J9_VM_FUNCTION_VIA_JAVAVM(javaVM, function) ((javaVM)->internalVMFunctions->function)
 #endif /* J9_INTERNAL_TO_VM */
 
+#define J9VTABLE_HEADER_FROM_RAM_CLASS(clazz) ((J9VTableHeader *)((J9Class *)(clazz) + 1))
+#define J9VTABLE_FROM_HEADER(vtableHeader) ((J9Method **) ((J9VTableHeader *)(vtableHeader) + 1))
```

Comment:
Both of these could use some extra brackets to isolate the cast from the +1, since either interpretation is syntactically valid, but have wildly different results.